src/OAuth/FortifiProvider.php

```diff
@@ -260,8 +260,13 @@ protected function getDefaultScopes()
    */
   protected function checkResponse(ResponseInterface $response, $data)
   {
-    if($response->getStatusCode() != 200)
+    $dataIsArray = is_array($data);
+    if($response->getStatusCode() != 200 || !$dataIsArray)
     {
+      if(!$dataIsArray)
```

Comment:
we should probably just return the default message if data is not an array.  we're just putting it into an array so we know the message key won't exist.